app/actions/remote/team.ts

```diff
@@ -355,3 +395,21 @@ export async function handleKickFromTeam(serverUrl: string, teamId: string) {
         logDebug('Failed to kick user from team', error);
     }
 }
+
+export async function getTeamMembersByIds(serverUrl: string, teamId: string, userIds: string[]) {
+    let client;
+    try {
+        client = NetworkManager.getClient(serverUrl);
+    } catch (error) {
+        return {error};
+    }
+
+    try {
+        const members = await client.getTeamMembersByIds(teamId, userIds);
```

Comment:
Same question about not storing the relation in the database, and more so, if the user is not present, why not adding it?

Reply:
Updated